slack_bolt/listener/async_internals.py

```diff
@@ -1,58 +0,0 @@
-from logging import Logger
```

Comment:
As this internal code is no longer used, we can safely delete the whole. We've switched to `slack_bolt/kwargs_inject/async_utils.py`